package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xng-breadcrumb-workspace",
-  "version": "10.0.2",
+  "version": "10.0.1",
```

Comment:
For version changes, one normally increments the version number.
So we'd expect to see 10.0.3